packages/slack-config-ui/slack_configure.js

```
@@ -0,0 +1,8 @@
+Template.configureLoginServiceDialogForSlack.helpers({
+  siteUrl: () => Meteor.absoluteUrl(),
+});
+
+Template.configureLoginServiceDialogForSlack.fields = () => [
+  {property: 'clientId', label: 'Client ID'},
```

ESLint

[error] Replace `property:·'clientId',·label:·'Client·ID'` with `·property:·'clientId',·label:·'Client·ID'·`

(prettier/prettier)